donation-ticker.html

```diff
@@ -46,7 +46,7 @@ <h1 class="h5">{{ entity.sumDonations | formatMoney }} / {{ entity.fundraisingGo
 		Vue.filter(
 			'formatDate',
 			(value) => {
-				return moment(value).format('MM/DD/YYYY hh:mm');
+				return dayjs(value).format(window.config.donationTicker.dateMask);
```

Comment:
@timmixell Something like this?

And in [config.js](https://github.com/DonorDrive/StreamKit/pull/7/files#diff-e22b50e740e102c8de5e858dbcd7753d):
```
donationTicker: {
        // ...	
        // Donation date mask/format
	dateMask: 'MM/DD/YYYY hh:mm a'   
},
```